react/src/index.js

```diff
@@ -82,6 +84,12 @@ Sentry.init({
           // name: window.location.pathname.replace(/\/employee.*/,'/employee/:id')
         };
       },
+      _experiments: {
+        // This enables tracing on user interactions like clicks
+        enableInteractions: true,
```

Comment:
Hmmm, I'll have to check that this doesn't break our custom-instrumented checkout transaction that has an error attached to it.